test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -798,11 +801,11 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 				},
 			}, ignoreLastChangeTime))
 
-			ginkgo.By("Finishing workload", func() {
```

Comment:
I guess we are not being very consistent, but the idea of putting things inside a function seems visually useful. I would just revert.

Reply:
I see, I did it for consistency, but have no issue to revert, so that the diff is cleaner.